content/governing-content/accountability-ownership/intro.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: intros/intro
-subtitle: Govern content to assign clear lines of accountability and ownership for leaders, doers and specialists.
+subtitle: When an agency governs content they assign clear lines of accountability and ownership.
 category: Governing content
 ---
-When everyone involved in content production knows who does what and how, teams are better able to work together and break down silos.
+This ensures everyone involved in the content lifecycle knows who does what, according to their expertise. Teams are better able to work together and break down silos.
```

Comment:
Possibly it would be good to start sentence with `Content governance ensures` in case people skip the abstract when scanning down the page?